### PR TITLE
[Debug] Added configuration reference for new debug options

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1433,6 +1436,24 @@ value.
...
+
+Use the application logger instead of the PHP logger for logging PHP errors.
+
+throw
+.....
+
+**type**: ``boolean`` **default**: ``%kernel.debug%``
+
+Throw PHP errors as ``\ErrorException`` instances. The parameter
+``debug.error_handler.throw_at`` control the threshold.
```
Comment (xabbuh) [inline reference/configuration/framework.rst]: controls

**Comment by xabbuh** [comment]
Please also add `versionadded` directives.

File: reference/configuration/framework.rst (hunk 1 above)
Comment (lyrixx) [inline reference/configuration/framework.rst]: thanks. Fixed.

**Comment by xabbuh** [comment]
👍 

Status: Reviewed

**Comment by wouterj** [comment]
Thanks @lyrixx, it looks perfect!

**Pull request merged**